minibroker/minibroker-master.yaml

```diff
@@ -0,0 +1,272 @@
+---
+shared:
+  helm-charts-s3-configs: &helm-charts-s3-configs
+    access_key_id: ((s3-access-key))
+    bucket: ((s3-bucket))
+    disable_ssl: ((s3-disable-ssl))
+    endpoint: ((s3-endpoint))
+    private: false
+    secret_access_key: ((s3-secret-key))
+  helm-charts-src-configs: &helm-charts-src-configs
+    branch: ((helm-charts-src-branch))
+    private_key: ((github-private-key))
+    uri: ((helm-charts-src-repo-ref))
+  s3-configs: &s3-configs
+    access_key_id: ((s3-access-key))
+    bucket: ((s3-bucket))
+    disable_ssl: ((s3-disable-ssl))
+    endpoint: ((s3-endpoint))
+    private: false
+    secret_access_key: ((s3-secret-key))
+  src-ci-configs: &src-ci-configs
+    uri: ((ci-repo))
+    branch: ((ci-branch))
+    private_key: ((github-private-key))
+
+
+resources:
+- name: src
+  type: git
+  source:
+    branch: ((src-branch))
+    private_key: ((github-private-key))
+    uri: ((src-repo-ref))
+
+- name: src-redis
+  type: git
+  source:
+    <<: *helm-charts-src-configs
+    paths:
+    - stable/redis
+
+- name: src-postgres
+  type: git
+  source:
+    <<: *helm-charts-src-configs
+    paths:
+    - stable/postgresql
+
+- name: src-mariadb
+  type: git
+  source:
+    <<: *helm-charts-src-configs
+    paths:
+    - stable/mariadb
+
+- name: src-mongodb
+  type: git
+  source:
+    <<: *helm-charts-src-configs
+    paths:
+    - stable/mongodb
+
+- name: src-ci
+  type: git
+  source:
+    <<: *src-ci-configs
+    paths:
+    - minibroker
+
+- name: docker.minibroker
+  type: docker-image
+  source:
+    repository: ((docker-org))/minibroker
+    username: ((docker-username))
+    password: ((docker-password))
+
+
+- name: s3.repo-index
+  type: s3
+  source:
+    <<: *s3-configs
+    versioned_file: ((s3-prefix))/index.yaml
+
+- name: s3.minibroker-chart
+  type: s3
+  source:
+    <<: *s3-configs
+    regexp: ((s3-prefix))/minibroker-(.*).tgz
+
+- name: s3.helm-charts-repo-index
+  type: s3
+  source:
+    <<: *helm-charts-s3-configs
+    versioned_file: ((helm-charts-s3-prefix))/index.yaml
+
+- name: s3.redis-chart
+  type: s3
+  source:
+    <<: *helm-charts-s3-configs
+    regexp: ((helm-charts-s3-prefix))/redis-(.*).tgz
+
+- name: s3.postgresql-chart
+  type: s3
+  source:
+    <<: *helm-charts-s3-configs
+    regexp: ((helm-charts-s3-prefix))/postgresql-(.*).tgz
+
+- name: s3.mariadb-chart
+  type: s3
+  source:
+    <<: *helm-charts-s3-configs
+    regexp: ((helm-charts-s3-prefix))/mariadb-(.*).tgz
+
+- name: s3.mongodb-chart
+  type: s3
+  source:
+    <<: *helm-charts-s3-configs
+    regexp: ((helm-charts-s3-prefix))/mongodb-(.*).tgz
+
+groups:
+- name: minibroker
+  jobs:
+  - build-minibroker-image
+  - generate-minibroker-chart
+  - generate-minibroker-repo
+- name: charts
+  jobs:
+  - generate-mariadb
+  - generate-mongodb
+  - generate-postgres
+  - generate-redis
+  - generate-repo
+
+jobs:
+- name: build-minibroker-image
+  serial: true
```

Comment:
This still needs serial? (It isn't in check)

Reply:
Fixed in 1c2860f